tests/by-util/test_env.rs

```diff
@@ -7,6 +7,7 @@
 #[cfg(target_os = "linux")]
 use crate::common::util::expected_result;
 use crate::common::util::TestScenario;
+#[rustfmt::skip]
```

Comment:
I'm very surprised that this is necessary, and can't reproduce it. In fact, all CI so far always was green, so this is not a systemic issue. Also, removing the leading `::` would break the code, so any program that suggests just removing them without replacement is obviously wrong.

Which rustfmt version are you using, and how exactly do you invoke it?

Reply:
please remove these skips :)

if needed, we will fix them into a different PR

Reply:
Huh, that is very weird.
```console
$ cargo +stable fmt --version # I have the same version as you
rustfmt 1.7.0-stable (07dca48 2024-02-04)
$ cargo +stable fmt --all --check # It does not report anything
$ cargo +stable fmt --all # And running it in the same manner …
$ git diff # … does not cause any file changes.
$
```

If the formatting issues cause pre-commit to fail, you can skip it completely via `git commit --no-verify`. However, that's just a temporary solution.